### PR TITLE
add 2 missing enchants, fix one id

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: db/professions/enchanting/skills.lua [hunk 1]
```diff
@@ -126,7 +126,7 @@ TRADE_SKILLS_DATA_TURTLE["skills"]["Enchanting"] =
             },
         },
         {
-            ["id"] = 1708151578,
+            ["id"] = 45071,
```
Comment (refaim) [inline db/professions/enchanting/skills.lua:129]: How do you find this ID?

https://database.turtle-wow.org/?item=51286 there is no link to spell here 
https://database.turtle-wow.org/?spell=45071 and there is no valid spell here

**Review by refaim — CHANGES_REQUESTED** [review]

**Comment by refaim** [comment]
Please update version history in README.md and addon version in TradeSkillsData-turtle.toc

File: db/professions/enchanting/skills.lua (hunk 1 above)
Comment (ghost) [reply db/professions/enchanting/skills.lua:129]: I noticed the ID showing up when scanning through my chars known skills but not being in the available table. Superwow's `SpellInfo()` function returns `Enchant 2H Weapon - Minor Intellect` as the name for that spell ID.

**Comment by ghost** [comment]
Added changes to readme and incremented the version in the .toc file.

**Comment by refaim** [comment]
Thanks!

**Pull request merged**